src/h3lib/lib/directedEdge.c

```diff
@@ -253,11 +247,11 @@ void H3_EXPORT(getH3UnidirectionalEdgeBoundary)(H3Index edge, GeoBoundary* gb) {
     FaceIJK fijk;
     _h3ToFaceIjk(origin, &fijk);
     int res = H3_GET_RESOLUTION(origin);
-    int isPentagon = H3_EXPORT(h3IsPentagon)(origin);
+    int pentagon = H3_EXPORT(isPentagon)(origin);
```

Comment:
This is always an issue with `isX` functions, that the output also seems like it should be called `isX`. I'd prefer a more specific name here, e.g. `originIsPentagon`, since we frequently use `pentagon` for a pentagon H3 index - though the easiest option is probably just to inline `H3_EXPORT(isPentagon)(origin)` in the `if` condition.

Is there a chance for subtle bugs here with `H3_EXPORT`, given our more generic names, or will this fail compilation?

Reply:
It will fail compilation, but only if the prefix is not set (which is the default).

I went with this name for the variable because it was easier to write a vim macro to change all instances across the source files automatically.

Reply:
I guess my point is partly that there's no way to change it _back_ now, because `pentagon` is widely used as a variable name for something else in other contexts. Up to you, but I find `pentagon` confusing here

Reply:
Maybe `int isPent = H3_EXPORT(isPentagon)(origin);`?